### PR TITLE
Update enterprise-storage-contents.md

### PR DESCRIPTION
**Pull request by pankajagrawal16**
Remove wording of "such as" to make it explicit of what exactly is stored in external storage

### Why:

Customers running GHES have generally have strict compliance requirements. In order to use actions, they need to use external storage and having terms like "such as" creates confusion if more than what is mentioned is stored in the external storage.

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by monjie110 — COMMENTED** [review]
Hoo

**Comment by cmwilson21** [comment]
And welcome to the community! 🎉 

In the meantime, if you'd like to find other ways to contribute, check out our [help wanted](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) section to find open issues you can work on. 👍

File: data/reusables/actions/enterprise-storage-contents.md [hunk 1]
```diff
@@ -1 +1 @@
-{% data variables.product.prodname_actions %} uses external blob storage to store data generated by workflow runs, such as workflow logs{% ifversion actions-caching %}, caches,{% endif %} and user-uploaded build artifacts.
+{% data variables.product.prodname_actions %} uses external blob storage to store data generated by workflow runs, workflow logs{% ifversion actions-caching %}, caches,{% endif %} and user-uploaded build artifacts.
```
Comment (mattpollard) [inline data/reusables/actions/enterprise-storage-contents.md:1]:
```suggestion
{% data variables.product.prodname_actions %} uses external blob storage to store data generated by workflow runs. Stored data includes workflow logs{% ifversion actions-caching %}, caches,{% endif %} and user-uploaded build artifacts.
```

**Review by mattpollard — DISMISSED** [review]
Thanks, @pankajagrawal16 ⚡ I'll make a small revision to this and then get it published.

**Review by mattpollard — APPROVED** [review]

**Comment by pankajagrawal16** [comment]
> Thanks, @pankajagrawal16 ⚡ I'll make a small revision to this and then get it published.

Thanks @mattpollard

**Pull request merged**